package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "devDependencies": {
+    "@symfony/webpack-encore": "^0.22.4",
+    "admin-lte": "~2.3.10",
+    "bootstrap": "^3.3",
+    "eonasdan-bootstrap-datetimepicker": "~3.1.3",
+    "bootstrap3-dialog": "^1.34",
+    "font-awesome": "^4.7",
+    "icheck": "^1.0",
+    "ionicons": "^4.5.5",
+    "jquery": "^2.2.3",
+    "jquery-form": "^4.0",
+    "jquery-slimscroll": "^1.3",
+    "jquery-ui": "^1.12.1",
+    "jquery.scrollto": "^1.4",
+    "masonry": "^0.0.2",
```

Comment:
Sure this are **dev**Dependencies? AFAIK only build tools are devDependencies

Reply:
Yea they are because you build it in devMode in production it is already a "compiled", minified version which works without the yarn and node.